api/config/config_local.json

```diff
@@ -207,6 +207,11 @@
     "userManagerBaseUrl": "https:\/\/workbench-dev.verily.com\/api\/user",
     "organizationId": "00a62244-0497-460f-8d55-dd1eee3ce975",
     "defaultPodId": "b9e49b23-72e4-4b41-8c2d-cb4c9f976019",
-    "exfilManagerServiceAccount": "workbench-exfil-manager@prj-d-1v-ucd.iam.gserviceaccount.com"
+    "exfilManagerServiceAccount": "workbench-exfil-manager@prj-d-1v-ucd.iam.gserviceaccount.com",
+    "adminBigQuery": {
+      "logProjectId": "all-of-us-workbench-test",
+      "bigQueryDataset": "vwb_admin_test",
+      "workspaceTable": "wsm_workspaces"
```

Comment:
nit, workspaceTable would be the same for all envs, Suggest to make those constant value in java code, along with all other potential tables. (there are many more coming)